keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -343,35 +343,86 @@ are properly reloading tokens by:
 1.  Add annotation to audit events for legacy and stale tokens including
     necessary information to locate problematic client.
 
+### Test Plan
+
+#### TokenRequest/TokenRequestProjection
+
+- Unit tests
+- E2E tests
+  - Projected jwt tokens are correctly mounted. (conformance test)
+  - The owner and mode of projected tokens are correctly set
+  - In-cluster clients work with Token rotation
+
+#### RootCAConfigMap
+
```

Comment:
Add a description of tests for this feature gate

Reply:
done